src/driver/driver.py

```diff
@@ -21,12 +21,12 @@ def driver_loop():
 
 def add_gps_coord(route: List[GpsCoord], coord: GpsCoord) -> List[GpsCoord]:
     """Adds a GpsCoord to the route"""
-    pass
+    return route.append(coord)
```

Comment:
list.append returns `none`. What we actually want is to return a new list with this, so what should be happening here is `route + [coord]`. In this case, we are using concatenation instead of appending.

This returns a new list without affecting our input.